geometry_msgs/Quaternion[2] datas